src/patternfly/components/Chip/chip.scss

```
@@ -12,31 +12,25 @@
   // modified chip
   --pf-c-chip--m-overflow--BackgroundColor: var(--pf-global--BackgroundColor--light-300);
   --pf-c-chip--m-overflow--PaddingLeft: 0;
-  --pf-c-chip--m-overflow--BorderColor: transparent;
   --pf-c-chip--m-overflow--BorderWidth: 0;
-  --pf-c-chip--m-overflow--hover--BorderColor: var(--pf-global--secondary-color--100);
-  --pf-c-chip--m-overflow--hover--BorderWidth: var(--pf-global--BorderWidth--sm);
 
   // modified chip button
-  --pf-c-chip--m-overflow--button--BorderRadius: var(--pf-global--BorderRadius--sm);
-  --pf-c-chip--m-overflow--button--BorderWidth: 0;
-  --pf-c-chip--m-overflow--button--BorderColor: transparent;
-  --pf-c-chip--m-overflow--button--PaddingRight: var(--pf-global--spacer--sm);
-  --pf-c-chip--m-overflow--button--PaddingLeft: var(--pf-global--spacer--sm);
-  --pf-c-chip--m-overflow--button--active--BorderWidth: var(--pf-global--BorderWidth--sm);
-  --pf-c-chip--m-overflow--button--active--BorderColor: var(--pf-global--secondary-color--100);
-  --pf-c-chip--m-overflow--button--focus--BorderWidth: var(--pf-global--BorderWidth--sm);
-  --pf-c-chip--m-overflow--button--focus--BorderColor: var(--pf-global--secondary-color--100);
+  --pf-c-chip--m-overflow--c-button--BorderRadius: var(--pf-global--BorderRadius--sm);
```

I wonder if you should go ahead and make this change?

```suggestion
  --pf-c-chip--m-overflow--c-button--BorderRadius: var(--pf-c-chip--BorderRadius);
```

If we had direction on whether we can combine variables, I might remove this variable entirely and just use `border-radius: var(--pf-c-chip--BorderRadius);` on the overflow button.

---

and do the same with the border-width and border-color. Just use the `pf-c-chip--Border*` vars as the values for the overflow button, and remove all of the `m-overflow` border vars.

---

and same with the `padding` vars - just keep `--pf-c-chip--PaddingLeft` and use that for the `m-overflow`'s left/right padding and `c-button`'s left/right padding.